src/bot/warden/warden.py

```diff
@@ -7,6 +7,7 @@
 from .time_range import TimeRange
 
 WORKING_DAYS = WeekDay.MONDAY, WeekDay.TUESDAY, WeekDay.WEDNESDAY, WeekDay.THURSDAY, WeekDay.FRIDAY
+FIRST_FOUR_DAYS_WEEKS = WeekDay.MONDAY, WeekDay.TUESDAY, WeekDay.WEDNESDAY, WeekDay.THURSDAY
```

Comment:
It doesn't make any sense to create this variable :)
I've created because it is `working days` - easy to understand and probably wide to use entity